package.json

```diff
@@ -194,4 +194,4 @@
     "./ja/modern.min": "./dist/ja-answers-modern.min.js",
     "./ja/template": "./dist/ja-answerstemplates.compiled.min.js"
   }
-}
\ No newline at end of file
+}
```

Comment:
could you try running npm install again? Just to make sure the package-lock needs the changes it has

Reply:
yup! no changes detected after npm i